bin/VMSlogger

```diff
@@ -341,7 +390,11 @@ async def main(args, pipe=None):
             sys.exit(1)
         file_type += "5"
         if args.size is None:
-            args.size = 86400000
+            if args.rotate is None:
+                args.size = 86400000
```

Comment:
Please consider to explain the unit here.

Reply:
size is unitless, it's number of records. But default choice explained.